### PR TITLE
Added Scrambled Eggs

### PR DESCRIPTION
**Pull request by 2digitman**
## About the PR

This adds scrambled eggs to the list of recipes so people can have them. As far as I know this shouldn't impact anything other than having to add it to the wiki.

**Media**

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tex
- add: Added scrambled eggs.

### CODE REVIEW

File: Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_meal.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@
       - FoodMealEggsbenedict
       - FoodMealOmelette
       - FoodMealFriedegg
+      - FoodMealScrambledEggs
```
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_meal.yml:32]:
```suggestion
      - FoodMealScrambledEggs # Delta V
```

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml [hunk 2]
```diff
@@ -566,6 +566,31 @@
...
+      - egg
+      - salty
+  - type: Sprite
+    layers:
+    - state: bowl
+    - state: scrambled-eggs
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 10
```
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml]:
```suggestion
        maxVol: 10
```
This is case-sensitive and causes errors.

File: Resources/Prototypes/Recipes/Cooking/meal_recipes.yml
Comment (Colin-Tel) [inline Resources/Prototypes/Recipes/Cooking/meal_recipes.yml:1]: This addition should be in a DeltaV directory, not touching an upstream file.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: This addition should be in a DeltaV directory, not touching an upstream file.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml [hunk 3]
```diff
@@ -566,6 +566,31 @@
...
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 9
+        - ReagentId: TableSalt
+          Quantity: 1
```
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:591]: Replace `TableSalt` with `Protein` (and adjust the amount, like 5 of each) as there is no need to add salt to a dish that tells you it's salty. This also means chefs won't be able to get a possible reimbursement on TableSalt.

**Review by Colin-Tel — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml
Comment (2digitman) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: I'm a little confused by what you mean. Where is the DeltaV directory and what is an upstream file?

Comment (Colin-Tel) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: `Resources/Prototypes/DeltaV` is a mirror directory where we put our specific files. The stuff you changed in this file doesn’t *have* to be here, so it shouldn’t touch an upstream file if it doesn’t have to. 

An upstream file is one that comes stock with the game. If you make a change to an upstream file, we ask that you include a comment that shows/explains the change.

Comment (2digitman) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: Ok I see. I'll close this and fix my issues.

**Comment by 2digitman** [comment]
Moving files to DeltaV directory.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml
Comment (Colin-Tel) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: You should keep the branch open if you’re going to make changes to it, unless you’re going to take a long time or something

**Comment by 2digitman** [comment]
Ready for Re-review

File: Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa",
```
Comment (Colin-Tel) [inline Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json:4]: Who drew the scrambled egg sprite? Is it also from tgstation and modified by Swept?

**Comment by Colin-Tel** [comment]
Wait, you did this on your master branch.

File: Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json (hunk 4 above)
Comment (2digitman) [reply Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json:4]: Nah my girlfriend made it. I didn't know what to put in the copyright. I can change it to her Alias if that is more appropriate.

Comment (Colin-Tel) [reply Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json:4]: Yeah you can put anything in the copyright field, whether it’s alias, URL, or both. Just make sure to include the preexisting copyright for the bowl.

Comment (2digitman) [reply Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json:4]: Got it. Which i am gonna guess is tgstation.

**Comment by 2digitman** [comment]
Please checkout when you can. Not sure if it is to your specifications yet. I changed the copyright.

**Pull request merged**